examples/showcase/360-gallery/index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>360&deg; Image Gallery</title>
+    <meta name="description" content="360&deg; Image Gallery - A-Frame">
+    <script src="https://aframe.io/releases/1.4.0/aframe.min.js"></script>
```

Comment:
link to master build instead of hardcoded version

Reply:
Can do.  Reason I did it this way was so that the HTML could easily be copy/pasted to another context (e.g. Glitch) and still work.

A small helping hand for beginners who might want to play with the example code, without cloning the A-Frame repo.

But if you prefer consistency with the other examples, that's - just wanted to share this consideration...